CMakeLists.txt

```diff
@@ -81,6 +81,13 @@ if (NOT MSVC)
     endif()
 endif()
 
+if(APPLE)
+    if(NOT DEFINED CMAKE_JS_VERSION)
+        # https://developer.apple.com/forums/thread/716965
+        add_cxx_flag_if_supported(-fno-objc-msgsend-selector-stubs)
+    endif()
+endif()
+
```

Comment:
We want/need this on the `realm-js-ios` target in `src/ios/CMakeLists.txt` specifically because that's the one that compiled Objective-C code (`platform.mm`). No need to define it globally.